Build fails because of changes in npm@5

```
[Success]  Base image pulled
[Info]     Starting Dockerfile build
[Build]    Step 1 : FROM resin/raspberrypi3-node:slim
[Build]     ---> 15b3ddada744
[Build]    Step 2 : WORKDIR /usr/src/app
[Build]     ---> Running in 9f88a0dfa634
[Build]     ---> 2d41387fc790
[Build]    Removing intermediate container 9f88a0dfa634
[Build]    Step 3 : COPY package.json package.json
[Build]     ---> 0fe5fcb88a8e
[Build]    Removing intermediate container 7a2888191de4
[Build]    Step 4 : RUN JOBS=MAX npm install --production --unsafe-perm && npm cache clean && rm -rf /tmp/*
[Build]     ---> Running in 75245dd8439c
[Info]     Still working...
[Build]    npm notice created a lockfile as package-lock.json. You should commit this file.
[Build]    added 42 packages in 25.594s
[Build]    npm ERR! As of npm@5, the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid. If you want to make sure everything is consistent, use 'npm cache verify' instead.
[Build]    npm ERR!
[Build]    npm ERR! If you're sure you want to delete the entire cache, rerun this command with --force.
[Build]    
[Build]    npm ERR! A complete log of this run can be found in:
[Build]    npm ERR!     /root/.npm/_logs/2017-06-14T10_45_23_144Z-debug.log
[Build]    Removing intermediate container 75245dd8439c
[Error]    Build failed: The command '/bin/sh -c JOBS=MAX npm install --production --unsafe-perm && npm cache clean && rm -rf /tmp/*' returned a non-zero code: 1
```

--- a/Dockerfile.template
+++ b/Dockerfile.template
@@ -2,7 +2,7 @@
 # see more about dockerfile templates here: http://docs.resin.io/deployment/docker-templates/
 # and about resin base images here: http://docs.resin.io/runtime/resin-base-images/
 # Note the node:slim image doesn't have node-gyp
-FROM resin/%%RESIN_MACHINE_NAME%%-node:slim
+FROM resin/%%RESIN_MACHINE_NAME%%-node:6-slim
 
 # use apt-get if you need to install dependencies,
 # for instance if you need ALSA sound utils, just uncomment the lines below.

